beater/handlers.go

```diff
@@ -36,6 +37,7 @@ const (
 	BackendErrorsURL        = "/v1/errors"
 	FrontendErrorsURL       = "/v1/client-side/errors"
 	HealthCheckURL          = "/healthcheck"
+	SourcemapsURL           = "/sourcemaps"
```

Comment:
This is not versioned. It might be fine, but could you explain the rationale briefly?